drivers/note/note_initialize.c

```diff
@@ -52,11 +52,11 @@
 
 int note_early_initialize(void)
 {
-  int ret = 0;
+  int ret = OK;
```

Comment:
but why relace 0 with OK? if possible, it's better to remove OK and ERROR from include/sys/types.h since the symbol confliction happen many time because include/sys/types.h has this non-standard but very common symbol.

Reply:
note_driver_register() will return OK, why not replace 0 to OK?
https://github.com/apache/nuttx/blob/master/drivers/note/note_driver.c#L1983

Reply:
@xiaoxiang781216 The usage of OK is what **you** have been insisting on before, you could check your previous review comments.

Reply:
> note_driver_register() will return OK, why not replace 0 to OK? https://github.com/apache/nuttx/blob/master/drivers/note/note_driver.c#L1983

If so, It's better to change OK to 0.


> @xiaoxiang781216 The usage of OK is what **you** have been insisting on before, you could check your previous review comments.

Where do I prefer OK/ERROR? If I suggest to use OK/ERROR before, I guess just because the original code already use OK/ERROR.

Reply:
Consistency is important. If there is no clear conclusion on the OK/ERROR, use OK is better than 0 in this case.

Reply:
yes, but this file use `0` and `< 0` consistently. Currently, both 0 and OK are used spread around the code base, if you want to reduce the consistency, it's better to convert OK/ERROR to 0/-Exxx.

Reply:
The scope of return value impact should not be constrained by the source file, should be return value from callee, right?

Reply:
Sorry, I don't catch what you are saying.

Reply:
Let's cut to the chase, I don't want to talk any more with you, either you replace all the OK/ERROR, or you merge this PR.